internal/jennies/golang/types.go

```diff
@@ -221,3 +259,20 @@ func (formatter *typeFormatter) formatIntersection(def ast.IntersectionType) str
 
 	return buffer.String()
 }
+
+func toOrderedMap(m map[string]interface{}) *orderedmap.Map[string, interface{}] {
```

Comment:
This function isn't specific to Go, so I would move it directly in the `orderedmap` package :)

Maybe as a kind of named constructor?

```go
func FromMap[K constraints.Ordered, V any](original map[K]V) *Map[K, V] {
	orderedMap := New[K, V]()

	keys := make([]K, 0, len(original))
	for key := range original {
		keys = append(keys, key)
	}

	sort.Slice(keys, func(i, j int) bool { return keys[i] < keys[j] })

	for _, k := range keys {
		orderedMap.Set(k, original[k])
	}

	return orderedMap
}
```